openreview/journal/invitation.py

```diff
@@ -2834,227 +3249,295 @@ def set_solicit_review_approval_invitation(self):
+                            'order': 9
                         },
                         'competing_interests': {
                             'value': {
-                                'type': 'string',
-                                'regex': '^[\\S\\s]{1,5000}$'
+                                'param': {
+                                    'type': "string",
+                                    'maxLength': 5000,
+                                    'input': 'textarea',
+                                }
                             },
                             'description': "Beyond those reflected in the authors' OpenReview profile, disclose relationships (notably financial) of any author with entities that could potentially be perceived to influence what you wrote in the submitted work, during the last 36 months prior to this submission. This would include engagements with commercial companies or startups (sabbaticals, employments, stipends), honorariums, donations of hardware or cloud computing services. Enter \"N/A\" if this question isn't applicable to your situation.",
-                            'order': 10,
-                            'readers': {
-                                'const': [ venue_id, paper_action_editors_id, paper_authors_id]
-                            }
+                            'order': 10
                         },
                         'human_subjects_reporting': {
                             'value': {
-                                'type': 'string',
-                                'regex': '^[\\S\\s]{1,5000}$'
+                                'param': {
+                                    'type': "string",
+                                    'maxLength': 5000,
+                                    'input': 'textarea',
+                                }
                             },
                             'description': 'If the submission reports experiments involving human subjects, provide information available on the approval of these experiments, such as from an Institutional Review Board (IRB). Enter \"N/A\" if this question isn\'t applicable to your situation.',
-                            'order': 11,
-                            'readers': {
-                                'const': [ venue_id, paper_action_editors_id, paper_authors_id]
-                            }
+                            'order': 11
                         }
                     }
                 }
             },
-            process=self.get_process_content('process/submission_revision_process.py')
-        )
+            'process': self.process_script                    
+        }
 
-        self.save_invitation(invitation)
+        self.save_super_invitation(self.journal.get_revision_id(), invitation_content, edit_content, invitation)
+
+    def set_note_revision_invitation(self, note):
+
+        return self.client.post_invitation_edit(invitations=self.journal.get_revision_id(),
```

Comment:
should this be `self.journal.get_revision_id(number=note.number)` or does the invitation `'TMLR/-/Revision'` exist?

Reply:
the super invitation is 'TMLR/-/Revision' and it is being used to create all the note revision invitation. 

the rule is:

super invitation: TMLR/-/[Name]
note invitation: TMLR/Paper[Number]/-/[Name]